examples/crhmc_sampling/README.md

```diff
@@ -16,6 +16,6 @@ For example: -DLP_SOLVE=/usr/lib/lpsolve/liblpsolve55.so
 
```

Comment:
Please add plots from the results from the various polytopes.

Reply:
Done.